Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1462,7 +1462,8 @@ class BrowserViewController: UIViewController {
 
     func openSearchNewTab(isPrivate: Bool = false, _ text: String) {
         popToBVC()
-        let engine = profile.searchEngines.defaultEngine
+        guard let engine = profile.searchEngines.defaultEngine else { return }
```

Comment:
```suggestion
        guard let engine = profile.searchEngines.defaultEngine,
            let searchURL = engine.searchURLForQuery(text) else { return }
```
We could add `if let searchURL = engine.searchURLForQuery(text)` to the guard and log the error in the body